### ISSUE TITLE
Tasks stats dont tally

### ISSUE THREAD

**Issue by planemad** [osmlab/to-fix#129]
In the [krakatoa task](http://osmlab.github.io/to-fix/?error=unconnected_major#/stats/rk) there are a total of 11,687 issues.

And while totaling the counts from every contributor:
- Edited: 13,138
- Fixed: 11,728
- Skipped 1,049
- Not an error: 558

The numbers don't seem to add up in any way to the total. The number of issues not fixed would be `total - fixed` which comes out to be negative. @tristen @aaronlidman some other way to compute this?

**Comment by aaronlidman** [comment]
Yeah I'm aware of this. It doesn't line up for a number of reasons, the main one being that when we update the data the old counts for each action is preserved but the total is reset. Which makes sense but I'm not sure how else to convey that.

This will require work around swapping out data: https://github.com/osmlab/to-fix-backend/issues/6

**Comment by planemad** [comment]
https://github.com/mapbox/data/issues/1053#issuecomment-123134117
